### ISSUE TITLE
Issue roundtripping multiline block quotes

### ISSUE THREAD

**Issue by DianaLease** [accordproject/markdown-transform#154]
When saving from slate to markdown and then re-rendering as Slate, multiline block quotes get lost. The first line is preserved, but the subsequent lines fall outside the block quote when we reload.

**Comment by jeromesimeon** [comment]
I was mistaken that this isn't a bug in `markdown-transform`. This even occurs for multi-line block quotes roundtrips on CommonMark:

```
bash-3.2$ ./packages/markdown-cli/index.js normalize --sample test.md 
./packages/markdown-cli/index.js normalize --sample test.md 
2:52:27 PM - info: 
test

> code
block

> with

more
than

one
line
test
bash-3.2$ cat test.md 
cat test.md 
test
> code
> block
> > with
> > 
> > more
> than
> 
> one
> line
test
```

**Comment by jeromesimeon** [comment]
There are more issues lurking in the markdown transform around blockquotes, which can in fact contain any markdown.

Here is a sample that shows issues with headings and lists:
```
bash-3.2$ cat ~/test.md
> This is a\
> linebreak.
> This is a quote.
>
> Heading Two
> ----
> This is more text.
> Ordered lists:
> 1. one
> 2. two
> 3. three
bash-3.2$ markus --version
0.9.6
bash-3.2$ markus normalize --sample ~/test.md 
12:08:00 PM - info: 
> This is a\
linebreak.
> This is a quote.
> Heading Two
----
> This is more text.
> Ordered lists:
1. > one
2. > two
3. > three
```

**Comment by jeromesimeon** [comment]
Should be properly fixed in #165 
```
bash-3.2$ cat test.md 
> This is a\
> linebreak.
> This is a quote.
>
> Heading Two
> ----
> This is more text.
> Ordered lists:
> 1. one
> 2. two
> 3. three
bash-3.2$ ../markdown-cli/index.js normalize --sample test.md
11:14:10 PM - info: 
> 
> This is a\
> linebreak.
> This is a quote.
> 
> Heading Two
> ----
> 
> This is more text.
> Ordered lists:
> 1. one
> 2. two
> 3. three
```

**Comment by jeromesimeon** [comment]
PR merged with the fix. The bigger test:

```
bash-3.2$ ../markdown-cli/index.js normalize --sample test/data/neestedblockquote.md 
8:44:21 AM - info: 
Once upon a time
> 
> This is a\
> linebreak.
> This is a quote.
> 
> Heading Two
> ----
> 
> This is more text.
> Ordered lists:
> 1. one
>    this has more in it
>    - nested list
>    - etc.
> 2. two
> 3. three
> 
> This should be a line
> 
> ---
> 
> This should be more text
> > 
> > This should be a nested blockquote
> > with more things in it
> > 
> > And a second paragraph in the nested blockquote
> 
> The end of the blockquote

The end
```
